zallet/tests/cmd/example_config.out/zallet.toml

```diff
@@ -101,6 +119,65 @@ network = "main"
 #orchard_actions = 50
```

Comment:
Move this to a `[builder.limits]` section, due to it being irrelevant to non-builder contexts.